types/lodash/ts3.1/lodash-tests.ts

```diff
@@ -6843,6 +6843,13 @@ fp.now(); // $ExpectType number
 // _.overEvery
 // _.overSome
 {
+    interface Named { name: string; }
+    interface Aged { age: number; }
+    const userDefinedTypeGuardNamed: <T>(item: T) => item is T & Named = anything;
+    const userDefinedTypeGuardAged: <T>(item: T) => item is T & Aged = anything;
+
+    _.overEvery(userDefinedTypeGuardNamed, userDefinedTypeGuardAged); // $ExpectType Refinement<{}, Named & Aged>
```

Comment:
```
ERROR: 6851:5  expect  TypeScript@next expected type to be:
  Refinement<{}, Named & Aged>
got:
  <T>(a: T) => a is T & Named & Aged
```

Reply:
Odd, I don't get that error. I do get this error however:

```ts
The 'use-default-type-parameter' rule threw an error in '/Users/oliverjash/Development/DefinitelyTyped/types/lodash/lodash-tests.ts':
TypeError: Cannot read property 'default' of undefined
    at checkArgsAndParameters (/Users/oliverjash/Development/DefinitelyTyped/node_modules/tslint/lib/rules/useDefaultTypeParameterRule.js:63:19)
    at cb (/Users/oliverjash/Development/DefinitelyTyped/node_modules/tslint/lib/rules/useDefaultTypeParameterRule.js:52:13)
    at visitNode (/Users/oliverjash/Development/DefinitelyTyped/node_modules/typescript/lib/typescript.js:16701:24)
    at Object.forEachChild (/Users/oliverjash/Development/DefinitelyTyped/node_modules/typescript/lib/typescript.js:16947:24)
    at cb (/Users/oliverjash/Development/DefinitelyTyped/node_modules/tslint/lib/rules/useDefaultTypeParameterRule.js:54:19)
    at visitNodes (/Users/oliverjash/Development/DefinitelyTyped/node_modules/typescript/lib/typescript.js:16710:30)
    at Object.forEachChild (/Users/oliverjash/Development/DefinitelyTyped/node_modules/typescript/lib/typescript.js:16936:24)
    at cb (/Users/oliverjash/Development/DefinitelyTyped/node_modules/tslint/lib/rules/useDefaultTypeParameterRule.js:54:19)
    at visitNodes (/Users/oliverjash/Development/DefinitelyTyped/node_modules/typescript/lib/typescript.js:16710:30)
    at Object.forEachChild (/Users/oliverjash/Development/DefinitelyTyped/node_modules/typescript/lib/typescript.js:16936:24)
```